src/main/java/com/playmonumenta/libraryofsouls/SpawnerInventory.java

```diff
@@ -146,26 +145,25 @@ public static void openSpawnerInventory(Soul soul, Player player, SoulsInventory
 		new SpawnerInventory(player, soul, item, previous).openInventory(player, LibraryOfSouls.getInstance());
 	}
 
+	/* Suppresses an error internal to Adventure in TextReplacementConfig.builder() */
+	@SuppressWarnings("NullAway")
```

Comment:
I wish there were a good way to limit suppressed warnings to just the relevant lines in case other errors sneak in; best I could come up with is to move them to their own methods.

Reply:
Yeah, this is annoying. Would like a more granular solution too but couldn't find one. 
I don't want to move the code out to a separate function though, that just makes it more awkward/harder to read. There's nothing wrong with the code, just this weird Adventure bug where their stuff is labelled wrong. It'll probably go away later with no code changes needed when we upgrade to 1.19 and bump the adventure version.